apps/www/src/lib/registry/default/ui/dialog/DialogScrollContent.vue

```diff
@@ -9,12 +9,20 @@ import {
   useEmitAsProps,
 } from 'radix-vue'
 import { X } from 'lucide-vue-next'
+import type { PointerDownOutsideEvent } from 'radix-vue/dist/DismissableLayer'
```

Comment:
Thanks @romanhrynevych ❤️ 

This issue should be fixed inside the `radix-vue` itself will create new issue for it

Also importing from `dist` is a bad idea


---

Let's keep this PR open until this issue resolved inside radix-vue

Reply:
Yeah, think that this will be better fixed inside radix-vue, but I think that it will be better to change from `mouse-down` to `click` event, reference: https://github.com/mui/material-ui/issues/15724

Maybe I will make PR inside `radix-vue`

Reply:
@sadeghbarati maybe you can help, receive so much such error when trying to run `docs:dev` script in `radix-vue` repo, there is no CONTRIBUTION file, maybe I do something wrong 🤔 

<img width="901" alt="image" src="https://github.com/radix-vue/shadcn-vue/assets/82209198/50eec00f-6f6a-43a3-9c7f-51cf5f536467">

Reply:
RadixVue package.json scripts are not aligned with their Vitepress doc

before you run Vitepress you need to build `radix-vue` in `packages/radix-vue`

Reply:
@romanhrynevych feel free to ask this question in Discord 


Discord ID: 254646464741900288

Reply:
removed this